CPDShell/main.py

```diff
@@ -17,17 +18,40 @@
 generated = ScipyDatasetGenerator().generate_datasets(Path(path_string), saver)
 data, expected_change_points = generated[distributions_name]
 
+print("Expected change points:", expected_change_points)
+
+# Graph algorithm demo
 graph_cpd = CPDShell(data)
 graph_cpd.scrubber.window_length = 150
 graph_cpd.scrubber.movement_k = 2.0 / 3.0
 
-print("Expected change points:", expected_change_points)
-
 res_graph = graph_cpd.run_cpd()
 res_graph.visualize(True)
 print("Graph algorithm")
 print(res_graph)
 
+
+# k-NN based algorithm demo
+def metric(obs1: float, obs2: float) -> float:
+    return abs(obs1 - obs2)
+
+
+K = 5
+THRESHOLD = 4.0
```

Comment:
Let's rename it to `KNN_THRESHOLD` (and I'll rename other thresholds with `BAYESIAN_` prefix in my PR). Just for readability reasons.